packages/opentelemetry-web/src/WebTracerProvider.ts

```diff
@@ -30,7 +28,7 @@ export interface WebTracerConfig extends TracerConfig {
   /**
    * plugins to be used with tracer, they will be enabled automatically
    */
-  plugins?: (BasePlugin<unknown> | InstrumentationBase)[];
+  plugins?: unknown[];
```

Comment:
why are you using `unknown` here ?

Reply:
because it doesn't matter what you put there it is deprecated now and you only see warning to use registerInstrumentations

Reply:
Then why have it at all?

Reply:
as I mentioned to raise a warning. People will still have old configuration, once they upgrade they will see warning. that is all, see line 43

Reply:
Agree that a two step process would be the way to go here